erpnext/hr/doctype/holiday_list/holiday_list.py

```diff
@@ -90,7 +90,38 @@ def is_holiday(holiday_list, date=today()):
 	"""Returns true if the given date is a holiday in the given holiday list
 	"""
 	if holiday_list:
-		return bool(frappe.get_all('Holiday List',
-			dict(name=holiday_list, holiday_date=date)))
+		return bool(frappe.get_all('Holiday List',dict(name=holiday_list, holiday_date=date)))
 	else:
 		return False
+
+def send_holiday_notification():
+	# holidays is a list of holidays which fall in 7 days from today
+	# new_holidays is a list of holidays which fall in 7 days from today and are not on a Saturday or Sunday
+	holiday_lists = frappe.get_all('Holiday List', ["name", "send_reminders_to"])
+	new_holidays = []
+
+	for holiday_list in holiday_lists:
+		today_date = today()
+		end_date = get_date_str(add_days(today(), 7))
+		holidays = frappe.get_all('Holiday',{"parent" : holiday_list.name, "holiday_date": ["BETWEEN", [today_date, end_date]]}, ["holiday_date","description"])
+
+		# Checking if a holiday falls on Saturday or Sunday
+		for holiday in holidays:
+			data = frappe.utils.get_weekday(holiday.holiday_date)
+			if data != 'Saturday' and data != 'Sunday':
```

Comment:
```suggestion
			if data != 'Saturday' and data != 'Sunday':
```
why is saturday and sunday hardcoded in here ?